src/components/home/home.html

```diff
@@ -154,8 +154,8 @@ <h2 class="font-weight-bold display-1">News and Features</h2>
             <v-btn block text-xs-center v-on:click="articleLoadMore(10)">Load More Stories</v-btn>
           </v-flex>
           <v-flex xs3 order-md3 order-xs3 color="red darken-4" text-xs-center>
-            <dailynewsletter/>
             <trendingpost/>
+            <dailynewsletter/>
```

Comment:
remove this change...
make **dailynewsletter** first, and then **trendingpost**

Reply:
Solve that issue.

Reply:
:+1: